### PR TITLE
Update treehouse autoplay

### PR DESCRIPTION
**Pull request by marcomilon**
Hello matt,

I've done some work on your extension.

1) Update the code to make it work with the current version of teamtreehouse. When video ends it will play the next video.
2) Remove autoplay of current video. With the current version of teamtrehouse it's not needed.
3) Remove fullscreen feature because it's not working anymore.
4) Add run_at=document_start to make the extension starts as soon as possible.

### CODE REVIEW

File: js/autoplay.js [hunk 1]
```diff
@@ -1,28 +1,25 @@
 $(function() {
 
 	// Set icon to active state
-	chrome.extension.sendMessage({active: "true", fullscreen: "fetch"}, function(response) {
-		// Full Screen support
-		if (response.fullscreen == "true") {
-			goFullscreen();
-		}
-	});
+	chrome.extension.sendMessage({active: "true"}, function(response) {
```
Comment (matt-west) [inline js/autoplay.js]: I think the response callback function is optional so we should be able to cut it here as it’s not doing anything. https://developer.chrome.com/apps/runtime#method-sendMessage

File: js/autoplay.js [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
+      setTimeout(function(){
+          var videoURL = $('.button.step-overlay-button.primary.icon-on-right').attr('href');
+          viewNextVideo(videoURL);
+      }, 5000);
+    });
 	
 	// Reset icon when we leave the page
 	$(window).unload(function() {
-		chrome.extension.sendMessage({active: "false"}, function(response) {});
+		chrome.extension.sendMessage({active: "false"}, function(response) {
```
Comment (matt-west) [inline js/autoplay.js]: Same as my last comment about cutting the response callback.

**Review by matt-west — CHANGES_REQUESTED** [review]
Thanks @marcomilon! I appreciate you jumping in to update this.

I just have a couple minor comments. Can you please make those updates and then I’ll release the new version to the Chrome Web Store.

**Comment by marcomilon** [comment]
Hello matt, 

I've made the changes requested to the PR. :+1:

**Review by matt-west — APPROVED** [review]

**Comment by matt-west** [comment]
Thanks @marcomilon!

**Comment by marcomilon** [comment]
No problem.

**Comment by matt-west** [comment]
I’ve released a new version of the extension to the Chrome Web Store.

**Comment by marcomilon** [comment]
Great, good to know.

:)

**Pull request merged**